engine/crates/engine-v2/src/sources/graphql/request/prepare.rs

```diff
@@ -97,7 +89,7 @@ impl PreparedFederationEntityOperation {
         );
         query.push_str("query");
         query.push('(');
-        write!(query, "${entities_variable_name}: [_Any!]!")?;
```

Comment:
Nitpicking a bit, but surely a few spaces for readability would be ok...

Reply:
fair enough